features/generate.feature

```diff
@@ -53,3 +53,11 @@ Feature: Generate a new project using Tuist
     Then I should be able to test the scheme ATests
     Then I should be able to build the scheme B
     Then I should be able to test the scheme BTests
+
+Scenario: The project is an iOS application with a target dependency and transitive framework dependency (ios_app_with_transitive_framework)
```

Comment:
I have to say it's nice to read this and see at a glance what we are covering with this test.